src/languages/en.ts

```diff
@@ -2401,6 +2402,25 @@ const translations = {
                 }
             },
         },
+        perDiem: {
+            subtitle: 'Set per diem rates to control daily employee spend.',
```

Comment:
Few strings like "Set per diem rates to control daily employee spend." and "Per diem" are repeated to match the UI. Wondering if it'd be easier to update later if we use one variable perhaps here in `common` and use it wherever required.